### PR TITLE
Set logging config/level and runtimeType via env variables

### LINKED ISSUE #14: Set logging config/level via env variables
From discussion [here](https://github.com/knative-sandbox/container-freezer/pull/4#discussion_r733638872): we'll want to pass logging configuration via configmaps --> envvar (similar to what [serving is doing](https://github.com/knative/serving/blob/main/cmd/queue/main.go#L139))

### PR DESCRIPTION
**Pull request by Myrat92**
# Changes
Set logging config/level and runtimeType via env variables

Fixes #14

### CODE REVIEW

**Comment by psschwei** [comment]
@Myrat92 Thanks for the PR! 

To fix the build failures, run `./hack/update-deps.sh` on your branch.

File: cmd/daemon/main.go [hunk 1]
```diff
@@ -17,9 +21,24 @@ import (
 
 const runtimeTypeContainerd string = "containerd"
 
+type config struct {
```
Comment (jwcesign) [inline cmd/daemon/main.go:24]: I think's it better to define it in pkg

**Comment by Myrat92** [comment]
cc to @psschwei

File: cmd/daemon/main.go (hunk 1 above)
Comment (psschwei) [reply cmd/daemon/main.go:24]: Why is that? Given that these values aren't used outside of main, I'd be inclined to leave them here, similar to how [serving does it](https://github.com/knative/serving/blob/ef89ff8061a935c44a283c07f872d47b57204089/cmd/queue/main.go#L67-L103).

Comment (jwcesign) [reply cmd/daemon/main.go:24]: make sense

**Review by psschwei — COMMENTED** [review]
Could you also add a way to set these variables via [configmap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#define-container-environment-variables-using-configmap-data)? Or alternatively add them to the [DaemonSet spec](https://github.com/Myrat92/container-freezer/blob/main/config/300-daemon-containerd.yaml#L23) if the configmap makes more sense for a separate PR.

**Comment by Myrat92** [comment]
Hello @psschwei 
I saw that `RUNTIME_TYPE` already exist in [DaemonSet spec](https://github.com/Myrat92/container-freezer/blob/main/config/300-daemon-containerd.yaml#L23).
`FreezerLoggingConfig` and `FreezerLoggingLevel` I think they could be empty, is it?  So I didn't add them to the env.

I will create a  new PR for setting these variables via configmap, do you think so?

**Comment by psschwei** [comment]
> Hello @psschwei I saw that `RUNTIME_TYPE` already exist in [DaemonSet spec](https://github.com/Myrat92/container-freezer/blob/main/config/300-daemon-containerd.yaml#L23). `FreezerLoggingConfig` and `FreezerLoggingLevel` I think they could be empty, is it? So I didn't add them to the env.

It's probably a bit of a nit, but I like adding them to the spec in this PR (even if the values are `""`) for completeness sake (i.e. we can set the values as a result of what was added in the PR)

> I will create a new PR for setting these variables via configmap, do you think so?

:+1:

**Comment by Myrat92** [comment]
cc @psschwei 
And I will create a new PR for setting these variables via configmap

**Review by psschwei — APPROVED** [review]
/lgtm
/approve

**Pull request merged**